root/static/scripts/common/entity.js

```diff
@@ -231,8 +231,14 @@ import formatTrackLength from './utility/formatTrackLength';
 
     class Label extends CoreEntity {
         selectionMessage() {
+            let code = this.toJSON().label_code;
+            if (code) {
+                code = ` (Label Code: LC-${"0".repeat(5 - String(code).length)}${code})`;
+            } else {
+                code = '';
+            }
             return ReactDOMServer.renderToStaticMarkup(
-                exp.l('You selected {label}.', {label: this.reactElement({target: '_blank'})}),
+                exp.l('You selected {label}{code}.', {label: this.reactElement({target: '_blank'}), code: code}),
```

Comment:
This is waaay too long a line - remember our limit is 78 chars. To be fair, it was already too long before, but still, might as well improve that :)
```suggestion
                exp.l(
                    'You selected {label}{code}.',
                    {
                        label: this.reactElement({target: '_blank'}),
                        code: codeSection,
                    },
                ),
```

Reply:
Sure. I copied your fix